Storage/Storage/CoreData/CoreDataManager.swift

```diff
@@ -48,34 +48,22 @@ public class CoreDataManager: StorageManagerType {
 
             /// Backup the old Store
             ///
+            var persistentStoreBackupError: Error?
```

Comment:
ugh yea I don't feel super good about having `persistentContainer: NSPersistentContainer` as a public lazy var, since it could be first triggered by anywhere (like the Reviews data source before). maybe this could be part of the effort to untangle Core Data setup from the app setup? https://github.com/woocommerce/woocommerce-ios/issues/2370